Dockerfile

```diff
@@ -4,12 +4,12 @@
 # Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
 #
 
-FROM registry.access.redhat.com/ubi9/openjdk-17:1.15 AS builder
+FROM registry.access.redhat.com/ubi9/openjdk-17:1.20 AS builder
 
 USER root
 WORKDIR /opt/kroxylicious
 COPY . .
-RUN ./mvnw -q -B clean package -Pdist -Dquick
+RUN mvn -q -B clean package -Pdist -Dquick
```

Comment:
cool, there's a maven ready to go in the openjdk image :man_shrugging:

Reply:
Yeah, you can check with https://catalog.redhat.com/software/containers/ubi9/openjdk-17/61ee7c26ed74b2ffb22b07f6?container-tabs=packages 3.8.5 is available.